README.md

```diff
@@ -571,6 +571,12 @@ jwt.request_formats = {
 
 By default, only requests without format are processed.
 
+#### token_header
+
+Request/response header which will transmit the JWT token. Make sure you configure this header as described in [Model configuration](#model-configuration)
+
+Defaults to 'Authorization'
```

Comment:
I was undecided between this and changing all references to 'Authorization' in the readme. I think this is less confusing but I am open to feedback